archive.njk

```diff
@@ -1,12 +0,0 @@
----
```

Comment:
The reason why deleting this file removes it from the navbar is due to the `eleventyNavigation`, that's how you get a link to show up in the navbar